internal/server/v1/page/place.go

```diff
@@ -30,7 +30,7 @@ func PlacePage(
 	ctx context.Context,
 	in *pb.PlacePageRequest,
 	store *store.Store,
-) (*pb.GetPlacePageDataResponse, error) {
+) (*pb.PlacePageResponse, error) {
 	node := in.GetNode()
 	if !util.CheckValidDCIDs([]string{node}) {
 		return nil, status.Errorf(codes.InvalidArgument, "Invalid entity")
```

Comment:
should we fail on empty category (eg line 39)?

Reply:
We could also default to overview in case of empty category

Reply:
Since this is at API level, it should be more direct. I would opt for return error if category is not set.

Updated that and removed the old cache in mixer as well.

Reply:
+1 for immediate failure